src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +74,10 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={(!title || !imgUrl || !imdbUrl || !imdbId) && true}
```

Comment:
```suggestion
            disabled={!title || !imgUrl || !imdbUrl || !imdbId}
```
а краще взагалі в змінну винести